app/models/user.rb

```diff
@@ -1,5 +1,6 @@
 class User < ApplicationRecord
   EMAIL_REGEX = Settings.VALID_EMAIL_REGEX
+  USER_PARAMS = :name, :email, :password, :password_confirmation
```

Comment:
```suggestion
  USER_PARAMS = [:name, :email, :password, :password_confirmation].freeze
```
để thế kia mà vẫn chạy được à